demos/apps/bookinfo/README.md

```diff
@@ -343,10 +341,12 @@ If you now refresh the `productpage` you'll see that while the load generator is
    $ ./cleanup.sh
    ```
 
+1. Optionally shut down the control plane services using the cleanup instructions [here](../../../kubernetes/README.md).
```

Comment:
This should be "replace", not "create", nor "apply.

Maybe this is why it is not working.

$ # (TODO) istioctl create -f mixer-rule-ratings-ratelimit.yaml
   $ kubectl apply -f ../../mixer-config-quota.yaml